testgrid/config.md

```diff
@@ -14,18 +14,18 @@ Testgrid is composed of:
 * A list of **dashboards**, or collections of dashboard tabs
 * A list of **dashboard groups** of related dashboards.
 
-Most of these objects are listed in [`config.yaml`]
+Most of these objects are simply listed in a [YAML config file][configuration] for Testgrid to consume.
 
 ## Prow Job Configuration
 
-If you just have a prowjob you want to appear in an existing dashboard, add annotations to that
-prowjob. You don't need to change a config file here.
+If you just have a [Prow job configuration](/config/jobs) you want to appear in an existing
+dashboard, add annotations to that Prow job. You don't need to change anything else.
 
-Simply add this to your prowjob:
+Simply add this to your Prow job:
 
 ```yaml
 annotations:
-  testgrid-dashboards: dashboard-name      # a dashboard defined in config.yaml.
+  testgrid-dashboards: dashboard-name      # a dashboard already defined in a testgrid configuration.
```

Comment:
Yeah here again I sort of feel like saying "config file" is a more explicit hint that it's a yaml file, but I'm not touching it if that's your style